try {
    somethingBadMightHappen();
  } catch (error) {
    console.error("Something bad happened");
    console.error(error);
  }